packages/components/src/components/icon/common/ic-selfie-verification-verified.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><path fill="var(--fill-color1)" fill-rule="evenodd" d="M5.854 6.44A1.5 1.5 0 0 1 6.914 6h2.172a1.5 1.5 0 0 1 1.06.44l1.415 1.414a.5.5 0 0 0 .353.146H13.5A1.5 1.5 0 0 1 15 9.5v6a1.5 1.5 0 0 1-1.5 1.5h-11A1.5 1.5 0 0 1 1 15.5v-6A1.5 1.5 0 0 1 2.5 8h1.586a.5.5 0 0 0 .353-.146l1.415-1.415Zm1.06.56a.5.5 0 0 0-.353.146L5.146 8.561A1.5 1.5 0 0 1 4.086 9H2.5a.5.5 0 0 0-.5.5v6a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 .5-.5v-6a.5.5 0 0 0-.5-.5h-1.586a1.5 1.5 0 0 1-1.06-.44L9.439 7.147A.5.5 0 0 0 9.086 7H6.914Z" clip-rule="evenodd"/><path fill="var(--fill-color1)" fill-rule="evenodd" d="M8 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Zm0 1a3 3 0 1 0 0-6 3 3 0 0 0 0 6Z" clip-rule="evenodd"/><circle cx="16" cy="4" r="4" fill="#4BB4B3"/><path fill="#fff" d="M14.39 3.922a.375.375 0 0 0-.53.53l1.125 1.126a.375.375 0 0 0 .53 0l2.625-2.625a.375.375 0 0 0-.53-.53l-2.36 2.36-.86-.86Z"/></svg>
```

Comment:
Most of the SVG's are same, we can reuse them, no need to create new files

Reply:
though icons looks similar, the status indicator(yellow, red, green) on each of those icons are different, making each svg unique.